src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,77 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final int FIGURE_NUMBER = 5;
+    private static final int MAX_NUMBER = 10;
+    private Random random = new Random();
+
+    public Figure getRandomFigure() {
+        String color = ColorSupplier.getRandomColor();

```

Comment:
ColorSupplier should be class-level
https://github.com/mate-academy/jv-oop-advanced/blob/master/checklist.md#think-about-which-variables-should-be-local-in-the-method-and-which-should-be-class-level

Reply:
not fixed